pandas/core/frame.py

```diff
@@ -5959,7 +5959,7 @@ def unstack(self, level=-1, fill_value=None):
 
         Returns
         -------
-        unstacked : DataFrame or Series
+        DataFrame or Series
```

Comment:
Super nitpicky but would be nice to order these the same way whenever used. A quick grep on codebase tells me `Series or DataFrame` appears more often so maybe stick with that ordering

@datapythonista something to consider